test/services/passwordRecovery/services/ChangePasswordService.test.js

```diff
@@ -0,0 +1,77 @@
+const chai = require('chai');
+const chaiAsPromised = require('chai-as-promised');
+const sinon = require('sinon');
+
+const { SilentError } = require('../../../../src/middleware/errors');
+const passwordRecovery = require('../../../../src/services/passwordRecovery/model');
+const ChangePasswordService = require('../../../../src/services/passwordRecovery/services/ChangePasswordService');
+
+chai.use(chaiAsPromised);
+
+describe('ChangePasswordService should', () => {
```

Comment:
Codacy found an issue: [Block must not be padded by blank lines.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5697327)